test/dynamo/test_error_messages.py

```diff
@@ -489,6 +489,8 @@ def fn(x):
     @scoped_load_inline
     @torch._dynamo.config.patch(inline_inbuilt_nn_modules=False)
     @unittest.skipIf(IS_FBCODE, "inline cpp_extension doesn't work in fbcode")
+    # Will fail until https://github.com/pybind/pybind11/issues/5774 is fixed.
+    @unittest.expectedFailure
```

Comment:
Is it practical for you to patch out `pybind11_detail_function_record_v`, to make this test pass.